src/components/Alert/index.tsx

```diff
@@ -117,20 +85,21 @@ type AlertProps = {
 }
 
 const Alert = ({
-  variant = 'standard',
   children,
-  iconSize = 32,
   icon,
   title,
   type = 'warning',
   className,
 }: AlertProps) => (
-  <StyledContainer type={type} variant={variant} className={className}>
-    <StyledIcon
-      name={icon || typesDefaultIcons[type]}
-      size={iconSize}
-      aria-hidden="true"
-    />
+  <StyledStackContainer
+    direction="row"
+    alignItems="center"
+    justifyContent="flex-start"
+    type={type}
+    className={className}
+    gap={2}
+  >
+    <Icon name={icon || typesDefaultIcons[type]} size={24} aria-hidden="true" />
```

Comment:
Should we really allow customisation of icon here ? 🤔

Reply:
@chambo-e you're right it is not a good behavior, I will remove it

Reply:
@chambo-e my bad I've read too fast, there might be use case where icon will be different especially for `info` variant

Reply:
example: ![Screenshot 2022-12-02 at 16 19 41](https://user-images.githubusercontent.com/15812968/205325989-2299bdf7-253f-4bf2-b32c-950d68b5357c.png)

Reply:
We need to define this with design, I think it will come later on with a bigger refactor for this component.

Reply:
Gotcha 

(omg that icon is awful)